src/Sulu/Bundle/ContactBundle/Entity/Address.php

```diff
@@ -341,11 +342,11 @@ public function getAddressType()
     /**
      * Set country.
      *
-     * @param \Sulu\Bundle\ContactBundle\Entity\Country $country
+     * @param string $country
      *
      * @return Address
      */
-    public function setCountry(\Sulu\Bundle\ContactBundle\Entity\Country $country = null)
+    public function setCountry($country)
```

Comment:
You know that we could use here (and in a lot other places probably) scalar type hints now?